__tests__/app.test.js

```diff
@@ -107,4 +107,67 @@ describe('GET Requests', () => {
                 })
         });
     });
+    describe('GET /api/articles/:article_id/comments', () => {
+        test('should return 200 and return an array of comments for the specified article', () => {
+            return request(app)
+                .get('/api/articles/1/comments')
+                .expect(200)
+                .then(({ body }) => {
+                    const { comments } = body;
+                    expect(comments.length).toBe(11);
+                    comments.forEach((comment) => {
+                        expect(comment).toMatchObject({
+                            comment_id: expect.any(Number),
+                            votes: expect.any(Number),
+                            created_at: expect.any(String),
+                            author: expect.any(String),
+                            body: expect.any(String),
+                            article_id: 1,
+                        });
+                    });
+                });
+        });
+        test('should return the comments in date/time order, with the most recent first', () => {
+            return request(app)
+                .get('/api/articles/1/comments')
+                .expect(200)
+                .then(({ body }) => {
+                    const { comments } = body;
+                    expect(comments[0]).toMatchObject({
+                        comment_id: 5,
+                        votes: 0,
+                        created_at: '2020-11-03T21:00:00.000Z',
+                        author: 'icellusedkars',
+                        body: 'I hate streaming noses',
+                        article_id: 1
```

Comment:
How are you testing for date order here when you're only testing for the first comment in the array?